src/social-media-icons/_base.scss

```diff
@@ -71,6 +77,76 @@
+		}
+	}
+
+	&.followus-vertical{
```

Comment:
This is dependant of the "followus" CSS class. Make it an accompanying CSS and move this block under "followus" CSS block. To use it, it will look like `<section class="gc-followus vertical">` and the corresponding CSS selector will be ".gc-followus.vertical"

Reply:
This has been done.